manifests/dev/vsphere-67u3/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -153,7 +153,7 @@ spec:
 apiVersion: v1
 data:
   "csi-migration": "false"
-  "csi-auth-check": "false"
```

Comment:
@lipingxue will get more detail on which vSpehre versions we have tested `csi-auth-check` feature, after we get confirmation, we can enable feature switch on those specific vSphere versions.

Reply:
Fixed. Disable for 67u3.